app/components/Banner/banner.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+const Banner: React.FC = () => {
+  return (
+    <div
+      className="w-full h-4/5 bg-cover bg-center flex justify-center items-center text-white"
+      style={{
+        backgroundImage:
+          'url("https://www.hrinasia.com/wp-content/uploads/2017/05/Asia-business-meetings.jpg")',
```

Comment:
download the image and save it into assets then use it from folder not from any external link